pom.xml

```
@@ -4,18 +4,66 @@
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<parent>
-		<groupId>life.qbic</groupId>
-		<artifactId>portlet-parent-pom</artifactId>
-		<version>3.1.3</version>
-	</parent>
-	<artifactId>Sample_Tracking_Status_Overview</artifactId>
+	<modules>
+		<module>sample-tracking-status-overview-app</module>
+		<module>sample-tracking-status-overview-domain</module>
+	</modules>
+	<artifactId>sample-tracking-status-overview</artifactId>
+	<groupId>life.qbic</groupId>
 	<version>0.1.0-SNAPSHOT</version> <!-- <<QUBE_FORCE_BUMP>> -->
 	<name>Sample Tracking Status Overview</name>
 	<url>http://github.com/qbicsoftware/Sample_Tracking_Status_Overview</url>
```

The actual repository url is seperated by hyphens not underscores
```suggestion
	<url>http://github.com/qbicsoftware/Sample-Tracking-Status-Overview</url>
```

---

its also written in lower case, i will correct it, thanks for pointing it out. I changed the name after creating it with qube and forgot to also change the url